Islandora-Devops/islandora_demo_objects
# Add "Service File" to the rows with tracks.

Workbench's --check feature balks if you try to load a media track (transcript) on anything except a Service File. This PR adds "Service File" to the media use column (keeping Original File; thus double-tagging) on the six rows that have media tracks.


To test: 
* Does the spreadsheet pass --check on workbench?
* When you visit those six objects, does the transcript show up as well as the video/audio?
* If you check the media listings for those objects, you should see one tagged as BOTH Original and Service file. There should not be an additional Service File created by derivatives.
